server/src/test/java/org/opensearch/index/translog/RemoteFsTranslogTests.java

```diff
@@ -131,6 +133,7 @@ public class RemoteFsTranslogTests extends OpenSearchTestCase {
     BlobStoreRepository repository;
 
     BlobStoreTransferService blobStoreTransferService;
+    Supplier<Version> minNodeVersionSupplier = () -> Version.CURRENT;
```

Comment:
We also need to test if minNodeVersion is less than the current version

Reply:
This Test class also tries to upload and download translog files using local fileSystem...Will take up this class tests once we introduce FS with metadata support.